library/src/main/scala/scala/scalajs/js/annotation/JSExportStatic.scala

```diff
@@ -23,7 +23,6 @@ import scala.annotation.meta._
  *
  *  @see [[https://www.scala-js.org/doc/interoperability/sjs-defined-js-classes.html Write JavaScript classes in Scala.js]]
  */
-@field @getter @setter
```

Comment:
Intentional? IIRC, this is only relevant for fields that are defined in the ctor of a class, so not really for statics. But in that case, shouldn't have been there in the first place...